### PR TITLE
Fix : added my BID and glassmorphism effect to card

### LINKED ISSUE #5: 1️⃣ Make your mark: Contribute to our project and help us create a platform for documenting life goals and aspirations!
Bellhop BellWe need more contributors to add their information to the JSON file and include two images that will then be seen on the Before I Die Code image gallery on the website. To contribute and start, first read the [README](https://github.com/BeforeIDieCode/BeforeIDieAchievements#readmel) and then begin following along on our [CONTRIBUTIONS-GUIDELINES](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md) file. Let's get after itMechanical Arm!!!!

### PR DESCRIPTION
**Pull request by Jothishwar**
## Proposed Changes and Benefits

- Added my Before I Die text with required images
- Added a glassmorphism effect to the card displaying the BID text of the contributors to improve the UI

## Screenshots

image

### Checklist

- [x] I have thoroughly read and understand [The Contribution Guidelines](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md).
- [x] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
  - `Feature` - adds new or amends existing user-facing behavior.
  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc.
  - `Fix` - bug fixes.
  - `Docs` - fixing typos and adding new content.
- [x] I have listed my proposed changes and their benefits.
- [x] I have linked the corresponding issue in the Issues section.

### CODE REVIEW

File: src/Components/MasonryLayout/MasonryBox/MasonryBox.module.css
Comment (XanderRubio) [inline src/Components/MasonryLayout/MasonryBox/MasonryBox.module.css:1]: Very neat, and great job adding this glass effect. It looks beautiful!

**Review by XanderRubio — APPROVED** [review]
HI @Jothishwar! It looks great and shows up great when viewing your work on the local dev server. We are currently having issues with dependencies affecting the preview link. I am going to see what happens when I merge your changes including the dependencies you added, as I recently deleted the package-lock.json, and this caused an issue that we now have ever since when attempting to run the preview deployment. I just wanted to let you know.

Thank you for taking the time to assist and add value to the UI, additionally sharing what you want to do before you die, and providing as much as you can for your parents.

**Comment by XanderRubio** [comment]
@Jothishwar I may need to revert this merge and address the dependency issue we are currently facing, but not the less we will get your contribution in the main branch.

**Comment by XanderRubio** [comment]
@Jothishwar, Your pull request has been successfully merged!!! Congratulations Bottle with Popping CorkBottle with Popping CorkBottle with Popping CorkDouble Exclamation MarkDouble Exclamation MarkDouble Exclamation Mark. I wish you all the best in your pursuit to live a life of purpose and giving back to your parents and traveling the world!

 If you haven't already, please go ahead and star Glowing StarGlowing StarGlowing Starthis repo and you are more than welcome to continually contribute to this project by assisting with our [open issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues), or adding your own new open issues, and additionally you can get an idea of ideas we have to continually grow this project further as technology and tech stack continually evolve by looking at the [ROADMAP](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/ROADMAP.md) for the project. Have a great day, and thank you for taking the time to contribute, JOTHISHWAR! You can view your contribution now on the [live link](https://before-i-die-achievements.vercel.app/). Have a great rest of your day, and I will be sending you an invite to be a member of the Before I Die Code now.

**Comment by XanderRubio** [comment]
I would like to apologize, @Jothishwar, for the delay and the unsmooth way we operate. I am sorry for not having shared the preview link with you earlier so that you could see your work. This happened because I made a mistake that led to dependency errors in the repository. This affected our ability to deploy, but it has been sorted out now. 

I want to express my gratitude for your contribution and enhancement of the UI. You are always welcome to contribute to this repository. However, I request that you create an issue first and wait to get assigned before creating your pull request. This is for organizational purposes. 

Thank you very much for your assistance and for helping to grow this project further! YOUR CONTRIBUTION CAN BE VIEWED ON THE [LIVE LINK NOW](https://before-i-die-achievements.vercel.app/)

Thank you,

Xander

**Comment by Jothishwar** [comment]
> I would like to apologize, @Jothishwar, for the delay and the unsmooth way we operate. I am sorry for not having shared the preview link with you earlier so that you could see your work. This happened because I made a mistake that led to dependency errors in the repository. This affected our ability to deploy, but it has been sorted out now.
> 
> I want to express my gratitude for your contribution and enhancement of the UI. You are always welcome to contribute to this repository. However, I request that you create an issue first and wait to get assigned before creating your pull request. This is for organizational purposes.
> 
> Thank you very much for your assistance and for helping to grow this project further! YOUR CONTRIBUTION CAN BE VIEWED ON THE [LIVE LINK NOW](https://before-i-die-achievements.vercel.app/)
> 
> Thank you,
> 
> Xander

@XanderRubio  Its okay, things like this happen once in a while. I'm happy that the issue is resolved now and my PR is merged. Looking forward to contribute more.

Thank you ✌️

**Comment by XanderRubio** [comment]
> > I would like to apologize, @Jothishwar, for the delay and the unsmooth way we operate. I am sorry for not having shared the preview link with you earlier so that you could see your work. This happened because I made a mistake that led to dependency errors in the repository. This affected our ability to deploy, but it has been sorted out now.
> > I want to express my gratitude for your contribution and enhancement of the UI. You are always welcome to contribute to this repository. However, I request that you create an issue first and wait to get assigned before creating your pull request. This is for organizational purposes.
> > Thank you very much for your assistance and for helping to grow this project further! YOUR CONTRIBUTION CAN BE VIEWED ON THE [LIVE LINK NOW](https://before-i-die-achievements.vercel.app/)
> > Thank you,
> > Xander
> 
> @XanderRubio Its okay, things like this happen once in a while. I'm happy that the issue is resolved now and my PR is merged. Looking forward to contribute more.
> 
> Thank you ✌️

Excellent!

**Pull request merged**